HelpAssistHer/help-assist-her
# Updated design for steps

## Description
This PR fixes the "steps" so that they aren't both showing at the same time, but instead Step 1 is showing and then once an option is chosen, Step 2 is shown. There is quite a bit of refactoring that had to happen to make this work so it might be easier to look at each of the commits separately.

## Test Plan
- Go to http://localhost:4000/mini-app/
- You should see this:
<img width="638" alt="screen shot 2019-02-09 at 8 02 18 pm" src="https://user-images.githubusercontent.com/6378435/52529385-b5a18c00-2ca5-11e9-86cd-f6d328c14838.png">

- Choose an option
- It should flip to this:
<img width="821" alt="screen shot 2019-02-09 at 8 02 45 pm" src="https://user-images.githubusercontent.com/6378435/52529459-b8e94780-2ca6-11e9-9293-54faeebe9d67.png">
